src/applications/vaos/utils/selectors.js

```diff
@@ -183,35 +186,13 @@ export function getChosenCCSystemId(state) {
   );
 }
 
-export function getRootOrganizationFromChosenParent(state, parentId) {
-  return getRootOrganization(
-    getParentFacilities(state),
-    parentId || getFormData(state).vaParent,
-  );
-}
-
-export function getRootIdForChosenFacility(state, parentId) {
-  const parentFacilities = getParentFacilities(state);
-
-  return getIdOfRootOrganization(
-    parentFacilities,
-    parentId || getFormData(state).vaParent,
-  );
+export function getRootIdForChosenFacility(state) {
+  const facility = getChosenFacilityInfo(state);
+  return facility?.id?.slice(0, 3);
```

Comment:
Is this right? Aren't all of the FHIR ids prefixed with `var`?

Reply:
Ah, you're right.  This selector is actually no longer used due to the changes in this PR.  Removing it entirely.